### ISSUE TITLE
Dashboard style updates

### ISSUE THREAD

**Issue by maya** [cloud-gov/cg-deploy-stratos#7]
Related to https://github.com/18F/design-lab/issues/28

Here are the styles I added to update the dashboard:
```
.mat-card {
    box-shadow: none;
}

body {
    background-color: #f5f5f3;
}

.boolean-indicator__container__icon-true.boolean-indicator__container__icon-subtle,
.dashboard__content a,
.mat-form-field.mat-focused.mat-accent .mat-select-arrow,
.mat-form-field.mat-focused.mat-primary .mat-select-arrow,
.mat-form-field.mat-focused .mat-form-field-ripple {
    color: #056dd4;
}

.mat-form-field.mat-focused .mat-form-field-ripple.mat-accent,
.mat-accent .mat-option.mat-selected:not(.mat-option-disabled),
.mat-primary .mat-option.mat-selected:not(.mat-option-disabled) {
    color: #056dd4;
}

.mat-toolbar.mat-accent,
.mat-toolbar.mat-primary,
.dot {
    background-color: #056dd4;
}

.tab-nav a {
    color: #fff;
}
```

There are probably ways to abstract this with variables and setting the primary colors, but since the app isn't setup to handle style changes locally, I'm unable to work in that way and posting the CSS here. /cc @rogeruiz

**Comment by mogul** [comment]
Note we now have a ton more info about testing and customizing Stratos locally from the presentation referenced in [this Slack thread](https://gsa-tts.slack.com/archives/C4XLJ1JBV/p1550685676000700).

TL;DR: They say to fork the stratos repository and add a `custom-src` directory with modifications in it, copying from the `examples/custom-src` as needed. Then run the front-end against a local Docker back-end. Here's a summary of how to do that:

![slide content showing how to run the Startos back end with a local front-end clone](https://user-images.githubusercontent.com/47762/53125169-27819d00-3512-11e9-89ec-d43fd8e2ada1.png)

**Comment by mogul** [comment]
I used that information to make a PR which cleans up some steps and enables better testing in https://github.com/18F/cg-deploy-stratos/pull/8

@maya regarding what you said in https://github.com/18F/design-lab/issues/28#issuecomment-455700264 : 
> When working on this issue, I found there wasn't a way to process the Sass / CSS files into the project when theming it.

Did you see these [instructions for customizing the theme](https://github.com/cloudfoundry-incubator/stratos/blob/v2-master/docs/customizing.md#customizing-the-theme
) by putting a `custom.scss` file in `custom-src/frontend/sass`?

**Comment by ccostino** [comment]
We're closing this given the recent changes we had to make, including forking Stratos itself, and updating our deployment to Stratos version 4.4.0.  We were able to migrate the existing customizations of the version 3.x deployment to the new one.
